File: solutions/beecrowd/1010/1010.js
const readline = require('readline')

const rl = readline.createInterface({
  input: process.stdin,
  terminal: false
});

(() => {
  let s = 0.0

  rl.on('line', (line) => {
    const [, a, b] = line.split(' ')

    s += parseInt(a) * parseFloat(b)
  })

  rl.on('close', () => {
    console.log(`VALOR A PAGAR: R$ ${s.toFixed(2)}`)
  })
})()

File: solutions/beecrowd/1010/1010.dart
import 'dart:io';

void main() {
  int b;
  double c, result = 0.0;
  var line;

  while ((line = stdin.readLineSync()) != null) {
    var numbers = line.split(' ');
    b = int.parse(numbers[1]);
    c = double.parse(numbers[2]);

    result += c * b;
  }

  print('VALOR A PAGAR: R\$ ${result.toStringAsFixed(2)}');
}
